File: examples/python/print_digits.py
"""Print digits for a language code."""
from __future__ import annotations

import sys

from worldalphabets import load_alphabet


def main() -> None:
    code = sys.argv[1] if len(sys.argv) > 1 else "en"
    script = sys.argv[2] if len(sys.argv) > 2 else None
    alphabet = load_alphabet(code, script=script)
    label = f"{code}-{script}" if script else code
    
    if alphabet.digits:
        print(f"{label} digits: {' '.join(alphabet.digits)}")
    else:
        print(f"{label}: No digits available")


if __name__ == "__main__":
    main()

File: examples/node/print_digits.js
/**
 * Print digits for a language code.
 */
const { getDigits } = require('../../index');

async function main() {
  const code = process.argv[2] || 'en';
  const script = process.argv[3];
  
  try {
    const digits = await getDigits(code, script);
    const label = script ? `${code}-${script}` : code;
    
    if (digits.length > 0) {
      console.log(`${label} digits: ${digits.join(' ')}`);
    } else {
      console.log(`${label}: No digits available`);
    }
  } catch (error) {
    console.error('Error:', error.message);
  }
}

main();
